src/components/highlighter/Highlighter.jsx

```diff
@@ -1,29 +1,34 @@
-// Repeated
+// Repeated styles
 const img = "bg-[url('/images/Highlighter.png')] bg-bottom bg-no-repeat"
-const text = "text-purple-light text-center font-bold tracking-[-1px]"
-const block = "block"
+const text =
+  "text-purple-light font-poppins text-center font-bold tracking-[-1px]"
 const lgSize = "lg:bg-[length:437.27px_38px] lg:text-[38px]"
```

Comment:
I suggest we avoid using pixels (px). Since they are not responsive and very hard to deal with when resolution changes and sizes change also. So we should be sticking with relative unit sizes like (rem unit which is relative to the root element = html, % unit which is relative to the parent). Hope that make sense.